org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/BuildManager.xtend

```diff
@@ -36,13 +37,18 @@ class BuildManager {
 
     val dirtyFiles = <URI>newLinkedHashSet
     val deletedFiles = <URI>newLinkedHashSet
+    List<IResourceDescription.Delta> unreportedDeltas = newArrayList 
     
     def Buildable submit(List<URI> dirtyFiles, List<URI> deletedFiles) {
     	queue(this.dirtyFiles, deletedFiles, dirtyFiles)
         queue(this.deletedFiles, dirtyFiles, deletedFiles)
         return [cancelIndicator|internalBuild(cancelIndicator)]
     }
 
+	/** 
+	 * @deprecated this method is no longer used
+	 */
+    @Deprecated
```

Comment:
show we throw a illegatestate/unsupportedmethod then?

Reply:
Don't think so. Breaking API at runtime feels bad.